src/analysis/special_functions/trigonometric.lean

```diff
@@ -2165,6 +2177,39 @@ end
 theorem sin_ne_zero_iff {θ : ℂ} : sin θ ≠ 0 ↔ ∀ k : ℤ, θ ≠ k * π :=
 by rw [← not_exists, not_iff_not, sin_eq_zero_iff]
 
+lemma tan_of_mul_pi_div_two (θ : ℂ) (h : ∃ k : ℤ, θ = (2 * k + 1) * π / 2) : tan θ = 0 :=
+by rw [tan, cos_eq_zero_iff.mpr h, div_zero]
+
+lemma tan_of_mul_pi (θ : ℂ) (h : ∃ k : ℤ, θ = k * π) : tan θ = 0 :=
```

Comment:
These two lemmas could also probably use better names.

Reply:
I think that we should have
```lean
lemma tan_mul_pi_div_two (k : ℤ) : tan (k * π / 2) = 0 := sorry

lemma tan_mul_pi (k : ℤ) : tan (k * π) = 0 := sorry
```